exonum-java-binding/app/src/main/java/com/exonum/binding/app/ServiceRuntimeBootstrap.java

```diff
@@ -50,7 +50,11 @@
    */
   private static final Stage APP_STAGE = Stage.PRODUCTION;
 
-  private static final ImmutableMap<String, Class<?>> DEPENDENCY_REFERENCE_CLASSES =
+  /**
+   * Dependency reference classes used to validate that service artifacts don't contain copies of
+   * the classes loaded by the application classloader.
```

Comment:
I'd clarify that those classes that are used in Exonum public APIs:

"the classes used in Exonum public APIs and loaded by the application classloader."

https://exonum.com/doc/version/0.12/get-started/java-binding/#using-libraries